app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -36,6 +36,7 @@ def prov_field_changed
           all_dialogs.each do |dialog_name, dialog|
             if dialog[:display] == :show && dialog_name == @edit[:new][:current_tab_key]
               page.replace_html(dialog_name, :partial => dialog_partial_for_workflow, :locals => {:wf => @edit[:wf], :dialog => dialog_name})
+              page << "$('[data-toggle=popover]').popovers()"
```

Comment:
Just to be sure, this line never does anything, since the function is called `popover`, not `popovers` .. are you sure you even need to initialize it manually?